notebooks/timelapse-camera-tutorial.ipynb

```diff
@@ -41,145 +41,40 @@
     "* Cameras were installed on the east and west side of the Grand Mesa, across a vegetation scale of 1-9, using the convention __XMR__:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    )
</pre><p>This will fail since the query returns &gt; 1000 rows. You can set <code>limit = 13371</code> (total number of rows returned in this case) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SnowEx/snow-cookbook/pull/28/discussion/#comment-2471207726'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>